### DISCUSSION TITLE
Anonymous authentication if client is in local subnet, mnesia otherwise

### DISCUSSION THREAD

**Discussion by weinrank** [emqx/emqx#5854] [Q&A] [answered]
Hi community,

is there a way to allow anonymous authentication for clients from specific (local) subnets and require (mnesia) authentication for everyone else?

Many thanks!

**Comment by zmstone** [comment, marked as answer]
Hi. @weinrank 
This is a feature for 5.0, where listeners can bind with a (chain of) authentication providers (mnesia included).
5.0 is still being developed though.
